src/alphafold_impact/pipelines/data_processing_nih_clinical_trials/README.md

```diff
@@ -0,0 +1,25 @@
+# NIH Clinical Trials Data Processing Pipeline (`data_processing_nih_clinical_trials`)
+
+## Overview
+The `data_processing_nih_clinical_trials` pipeline processes raw data fetched from `clinicaltrials.gov`. This pipeline focuses on transforming the raw data into format suitable for further analysis within the project.
+
+## Modules
+### `nodes.py`
+This module contains the core functionalities for the pipeline:
+- `trials_with_references`: Loads NIH clinical trials from partitioned dataset, filters out trials that do not reference a paper, concatenates results into one DataFrame.
+- `trials_links_to_papers`: Produces a DataFrame which can be used to link NIH clinical trials to research papers via PubMed ID or DOI.
```

Comment:
What wasn't obvious to me until I ran the code was that this has as input the output of `trials_with_references`. Maybe worth mentioning it for legacy sake.

Reply:
I've added a note regarding this